### PR TITLE
[Merged by Bors] - doc(Topology/PartitionOfUnity): add missing documentation

### PR DESCRIPTION
**Pull request by grunweg**
---

Found while cleaning up old branches, as you can see from the individual commits: it's much better to simply review the aggregate diff :-)

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Topology/PartitionOfUnity.lean [hunk 1]
```diff
@@ -119,10 +124,15 @@ every open covering `U : Set (Set X)` of `s` there exists a `BumpCovering` of `s
 subordinate to `U`.
 -/
 structure BumpCovering (ι X : Type*) [TopologicalSpace X] (s : Set X := univ) where
+  /-- The collections of continuous functions underlying this bump covering -/
   toFun : ι → C(X, ℝ)
+  /-- the supports of the underlying functions are a locally finite family of sets -/
   locallyFinite' : LocallyFinite fun i => support (toFun i)
+  /-- the functions are non-negative -/
   nonneg' : 0 ≤ toFun
+  /-- the functions sum up to at most one -/
```
Comment (j-loreaux) [inline Mathlib/Topology/PartitionOfUnity.lean:133]:
```suggestion
  /-- the functions are each at most one -/
```

File: Mathlib/Topology/PartitionOfUnity.lean [hunk 2]
```diff
@@ -97,10 +97,15 @@ that for every open covering `U : Set (Set X)` of `s` there exists a partition o
...
 structure PartitionOfUnity (ι X : Type*) [TopologicalSpace X] (s : Set X := univ) where
+  /-- The collection of continuous functions underlying this partition of unity -/
   toFun : ι → C(X, ℝ)
+  /-- the supports of the underlying functions are a locally finite family of sets -/
   locallyFinite' : LocallyFinite fun i => support (toFun i)
+  /-- the functions are non-negative -/
   nonneg' : 0 ≤ toFun
+  /-- the functions sum up to one on `s` -/
   sum_eq_one' : ∀ x ∈ s, ∑ᶠ i, toFun i x = 1
+  /-- the functions sum up at most one, globally -/
```
Comment (j-loreaux) [inline Mathlib/Topology/PartitionOfUnity.lean:108]:
```suggestion
  /-- the functions sum up to at most one, globally -/
```

**Comment by j-loreaux** [comment]
bors d+

**Comment by grunweg** [comment]
Good catches, and thanks for the quick review!
bors merge

**Pull request closed without merging**